### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Code analysis fixes and small efficiency fixes.

### CODE REVIEW

File: z3c/dependencychecker/dependencychecker.py [hunk 1]
```diff
@@ -501,9 +514,9 @@ def imports_from_doctests(path):
     for path, dirs, files in os.walk(path):
         for filename in [os.path.abspath(os.path.join(path, filename))
                          for filename in files
-                         if fnmatch.fnmatch(filename, '*.txt')
-                         or fnmatch.fnmatch(filename, '*.rst')
-                         or fnmatch.fnmatch(filename, '*.py')]:
+                         if fnmatch.fnmatch(filename, '*.txt') or
+                         fnmatch.fnmatch(filename, '*.rst') or
```
Comment (reinout) [inline z3c/dependencychecker/dependencychecker.py]: Is this pep8-suggested? I like the original a bit more, visually, but I know that "nice visual effect" isn't always the best reason :-)

**Comment by reinout** [comment]
Looks OK. A bit of cleanup is nice :-)

I've added one question in the PR.

File: z3c/dependencychecker/dependencychecker.py (hunk 1 above)
Comment (mauritsvanrees) [inline z3c/dependencychecker/dependencychecker.py]: Yes.  the pep8 checker says: `W503 line break before binary operator`.
Ah, that is a recent change that is added to the pep8 guidelines (but not visible on the official pep yet).
See the links in this issue: https://github.com/PyCQA/pycodestyle/issues/498

Comment (reinout) [inline z3c/dependencychecker/dependencychecker.py]: So.... "I was right all along"? :-)

Comment (mauritsvanrees) [inline z3c/dependencychecker/dependencychecker.py]: Well... you were right all along, but not anymore. :-) My change follows the new rules.

Comment (reinout) [inline z3c/dependencychecker/dependencychecker.py]: You're sure? If I read https://hg.python.org/peps/rev/3857909d7956, it seems like you should break before the binary operator. If the break is before the "or/and", then the "or/and" is the first one on the line, right?

Comment (mauritsvanrees) [inline z3c/dependencychecker/dependencychecker.py]: Argh, I am misreading it. I have reverted (removed) this commit.

BTW, the pep8 has been renamed to pycodestyle.

**Comment by reinout** [comment]
Maurits: I added you as collaborator, btw.

**Pull request merged**